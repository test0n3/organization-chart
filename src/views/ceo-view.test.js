import React from "react";
import { render } from "@testing-library/react";
import CeoName from "./ceo-view";

test("CeoName component", () => {
  const { asFragment } = render(<CeoName />);

  expect(asFragment()).toMatchSnapshot();
});
