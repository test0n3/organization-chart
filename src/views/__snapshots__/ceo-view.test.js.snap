// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CeoName component 1`] = `
<DocumentFragment>
  <div
    class="css-1tgedd4-CeoName"
  >
    <div>
      <form
        class="css-xctk3t-InputNameCeo"
      >
        <input
          class="css-lrjsfw-InputNameCeo"
          name="ceo"
          placeholder="Ceo's Name"
          type="text"
        />
      </form>
    </div>
  </div>
</DocumentFragment>
`;
